src/main/java/org/spongepowered/api/plugin/PluginContainer.java

```diff
@@ -59,4 +59,12 @@
      */
     Object getInstance();
 
+    /**
+     * Gets if this PluginContainer wraps something that isn't a 'real' plugin.
```

Comment:
What does 'real' mean? I would suggest 'loaded' or 'initialized' or something like that, since that's the big difference